### PR TITLE
[Documentation] Updated minimum RAM requirements

### PR DESCRIPTION
**Pull request by jaredmullane**
Signed-off-by: Jared Mullane <jmullane@fb.com>

## Summary

Updating the minimum RAM requirements for v1.5 and the product specifications document based on performance testing results from the lab. Changing minimum RAM from 2GB to 4GB. 

## Test Plan

Screen Shot 2021-04-30 at 1 16 59 PM

Screen Shot 2021-04-30 at 1 17 32 PM

### CODE REVIEW

**Review by hcgatewood — APPROVED** [review]
Change looks good

An aside: I understand the desire for PDFs for e.g. sharing with partners, but the majority of the content of that PDF (outside the appendix) should be reflected in the Docusaurus rather than siloed in a PDF

**Comment by ekowtaylor** [comment]
@jaredmullane Once this PR has been approved and passing all CI checks, use the ready2merge label to indicate the maintainers can merge your PR.

**Pull request merged**
